### DISCUSSION TITLE
Quick Q: Why isn't (or can't) --lowvram or --medvram be impletmented as a UI setting?

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#849] [Q&A] [answered]
Is there a reason why `--lowvram` or `--medvram` (or even `--theme dark`) isn't implemented in as a UI Setting?

**Comment by vladmandic** [comment, marked as answer]
it could, its just that its typically used very early during startup before settings are even loaded, so i decided not to for now.

**Reply by brknsoul** [reply]
I guess. You could write a routine that just dumps a setting to a file that launch.py could read early on and adjust the vram setting there.. but I guess that more hassle than it's worth. ;-)

Thanks for the answer.
